oneview/resource_logical_interconnect_group.go

```diff
@@ -651,79 +724,6 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 					},
 				},
 			},
-			"port_flap_settings": {
```

Comment:
Is the swapping of ```port_flap_settings``` up an down made intentionally..? Just asking because I didn't find any changes between the removed lines and added lines.